File: backend/src/main/java/com/skewwhiffy/auraltester/helper/CollectionHelper.java
package com.skewwhiffy.auraltester.helper;

import java.security.SecureRandom;
import java.util.List;

public class CollectionHelper {
    private final static SecureRandom random = new SecureRandom();
    private CollectionHelper() {
    }

    public static <T> T oneOf(List<T> source) {
        return source.get(random.nextInt(source.size()));
    }
}

File: backend/src/main/kotlin/com/skewwhiffy/auraltester/helper/CollectionHelper.kt
package com.skewwhiffy.auraltester.helper

import java.security.SecureRandom

private val random = SecureRandom()

fun <T> oneOf(source: List<T>) = source[random.nextInt(source.size)]

fun oneOf(source: IntRange) = oneOf(source.toList())

fun oneOf(source: String) = oneOf(source.toList()).toString()
